wiki/Twitter/fr.md

```diff
@@ -0,0 +1,10 @@
+## Nous tweetons aussi!
+
+osu! a plusieurs comptes twitter, chacun sert à quelque chose de différent. La plupart ne tweetent que peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
```

Comment:
Oh, I know why it was bothering me. In this case, "peu" is used like a "nom commun", so it needs an "article" ("un") to construct a correct sentence.